analyses/fusion-summary/01-fusion-summary.Rmd

```diff
@@ -67,10 +91,14 @@ prepareOutput <- function(fuseDF, bioid) {
 
 ```{r}
 dataDir <- file.path("..", "..", "data")
+fusDir <- file.path("..", "..", "analyses", "fusion_filtering", "results")
+annotDir <- file.path("..", "..", "analyses", "fusion_filtering", "references")
 #' The putative oncogenic fusion file is what we'll use to check for the 
+#' annotation file
+annot <- read.delim2(file.path(annotDir, "genelistreference.txt"), sep = "\t", header = T)
```

Comment:
If we don't change to readr::read_tsv(), we will need to add one more argument here: 
```suggestion
annot <- read.delim2(file.path(annotDir, "genelistreference.txt"), sep = "\t", header = TRUE, stringsAsFactors = FALSE)
```